docs/src/solutions.md

```diff
@@ -119,6 +119,12 @@ else
 end
 ```
 
+!!! warning "Querying after modification"
+    If a solved model is modified, then querying the solution is undefined
+    behaviour. Adding, deleting, or modifying a constraint (or variable)
+    may invalidate the any part of the solution (e.g., the values of
```

Comment:
"invalidate the any" should be replaced by "invalidate any". You can probably also remove the "(e.g., ...)". "Any part" is descriptive enough.